pkgs/development/tools/apksigcopier/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, installShellFiles
+, bash
+, pandoc
+}:
+
+# FIXME: how to "recommend" apksigner like the Debian package?
+
```

Comment:
```suggestion
```

Recommend in which way? Nixpkgs either installs something or it doesn't. The only "recommend" we have is the longDescription.

Reply:
I know. Which I why I'm not sure what the best alternative is. If there was a separate `apksigner` package, I'd depend on that. But since there isn't, I certainly don't want to depend on an entire Android SDK package.

Reply:
Can we use apktool?

Reply:
apktool doesn't support verifying signatures.

Reply:
and even if it did, it would be best to use `apksigner`, since that's the official tool.

Reply:
I *might* look into packaging `apksigner` separately, like Debian does. But not any time soon.

Reply:
We could also copy it into a separate output which gets cached by hydra.

Reply:
See #131258